### PR TITLE
move EndSessionReaction to core

### PR DESCRIPTION
**Pull request by veptechno**
Many channels can end session. But end session reaction is locked inside JAICP module. This moves EndSessionReaction to core and unifies the reaction to all channels.

### CODE REVIEW

File: channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaReactions.kt [hunk 1]
```diff
@@ -97,11 +98,14 @@ class AlexaReactions(
         response.builder.addAudioPlayerStopDirective()
     }
 
-    fun endSession(text: String? = null) {
-        text?.let { say(it) }
+    fun endSession(text: String): EndSessionReaction {
```
Comment (morfeusys) [inline channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaReactions.kt:101]: Why did you change the method's signature? `text` arg should be nullable for multi-channel scenarios where some content could be generated for many channels before the session ending.

**Review by morfeusys — CHANGES_REQUESTED** [review]

File: channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaReactions.kt (hunk 1 above)
Comment (veptechno) [reply channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaReactions.kt]: Ok

**Comment by Denire** [comment]
I suggest also moving `NewSessionReaction` from JAICP package somewhere close to `EndSessionReaction` as they belong close to each other.

**Review by Denire — APPROVED** [review]
